app/themes/user-admin/src/app/users-event-log/users-event-log.component.spec.ts

```diff
@@ -0,0 +1,271 @@
+import {
```

Comment:
need to add this line to prevent ts-list from freaking out: ` /* tslint:disable:no-unused-variable */`

this needs to be the VERY first line